tensorflow_datasets/core/download/downloader_test.py

```diff
@@ -128,8 +129,21 @@ def test_bad_http_status(self):
   def test_kaggle_api(self):
     fname = 'a.csv'
     with testing.mock_kaggle_api(filenames=[fname, 'b.txt']):
-      promise = self.downloader.download('kaggle://some-competition/a.csv',
-                                         self.tmp_dir)
+      # Testing Competition Downloader
+      promise = self.downloader.download(
```

Comment:
```
promise = self.downloader.download(
          'kaggle://competition/some-competition/a.csv'
```

Instead of using `kaggle.KaggleFile._COMPETITION_TYPE.prefix`

The reason is that the tests should check how the feature will be used in practice. Users won't use `kaggle.KaggleFile._COMPETITION_TYPE.prefix`.

Reply:
In practice one doesn't need to specify anything.
One can just put
`download_manager.download_kaggle_data("competition_name")`
or
`download_manager.download_kaggle_data("author_name/dataset")`

All this prefix and prefix-related things are mainly for internal use.

Reply:
The user can do `dl_manager.download('kaggle://competition/...')` so it seems to be user facing to me.

Reply:
Oh, I see. Thanks for pointing that out. :)